README.md

```diff
@@ -242,6 +242,21 @@ Examples for exec commands can be found in the [scripts](/scripts/exec) director
 | :------------------ | :--------------------------------------------- |
 | statement/remove    | Removes assignment, increment, decrement and expression statements. |
 
+## Config file
+
+There is a configuration file where you can fine-tune mutation testing.
+The config will be filled in YAML format.
+It can be connected using the "--config" parameter
```

Comment:
If `--config` is presented, the library will use the given config. Otherwise, no default config file will be used.